src/app/about/about.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss']
+})
+export class AboutComponent implements OnInit {
+
+  constructor() { }
+
+  aboutWho = 'EvalAI is built by a team of open source enthusiasts working\
```

Comment:
Can you make one constant file for all these constant string where we can put all description, title etc? @live-wire

Reply:
Hey @aka-jain, I moved this text to the HTML file since this text won't be reused anyway. Moreover texts like this could contain `<a>` tags which can't be stored as strings in a separate file :)